mk-poetry-dep.nix

```diff
@@ -61,6 +61,8 @@ pythonPackages.callPackage
               inherit pythonPackages pyProject;
             } else [ ];
 
+      pname = normalizePackageName name;
+      preferWheel' = preferWheel && pname != "wheel";
```

Comment:
Is the tick variable idiomatic? It's pretty hard to notice when reading the code, at least. Would it be easier to read as something like `preferWheelExceptSelf` or `resolvedPreferWheel`? (Anyone know a good single word to say "non-self-referencing"?)

Reply:
It's an FP thing and is read as "prefer wheel prime" and is idiomatic for internal-only variables.